### PR TITLE
Completed admin module

### PR DESCRIPTION
**Pull request by yashodgayashan**

### CODE REVIEW

File: frontend/web/src/components/home/body/dashboard/admin_dash/grid/view_request/view_request.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+const View_request = props => {
+  let history = useHistory();
+  const [
+    ,
+    ,
+    ,
+    ,
+    ,
+    ,
+    ,
```
Comment (deshankoswatte) [inline frontend/web/src/components/home/body/dashboard/admin_dash/grid/view_request/view_request.js:14]: What are these?

File: frontend/web/src/App.js [hunk 2]
```diff
@@ -2,20 +2,22 @@ import React, { Fragment } from "react";
...
     </Fragment>
   );
 }
 
 export default App;
 
-
 // const position = [51.505, -0.09]
 // const map = (
 //   <Map center={position} zoom={13}>
```
Comment (deshankoswatte) [inline frontend/web/src/App.js:23]: Commented code?

File: frontend/web/src/components/home/body/dashboard/admin_dash/grid/view_request/view_request.js (hunk 1 above)
Comment (yashodgayashan) [reply frontend/web/src/components/home/body/dashboard/admin_dash/grid/view_request/view_request.js:14]: This is array destucturing

File: frontend/web/src/App.js (hunk 2 above)
Comment (yashodgayashan) [reply frontend/web/src/App.js:23]: Not mine

**Review by deshankoswatte — APPROVED** [review]

**Pull request merged**